docs/servo_power_test/index.html

```diff
@@ -17,7 +17,7 @@
     <button id='request-serial-button'>Start</button>
     <button id='halt-button'>Cancel</button>
     <button id='download-button'>Download</button>
-    <span>number of iterations: <input id="iteration-input" type="number" placeholder="1"></span>
+    <span>number of iterations: <input id="iteration-input" type="number" value="1"></span>
```

Comment:
How about setting the default number of iteration to 1 instead of asking users to manually input it every time?

Reply:
I think it already sets the default number to 1. But my understanding is incorrect...?

Reply:
aaaaaaaaah, sorry somehow I saw the change in the opposite way... (like replacing `value` with `placeholder`, but actually not...)